jsapp/js/components/dataAttachments/connectProjects.es6

```diff
@@ -0,0 +1,561 @@
+
+          {(!this.state.isInitialised || this.state.isLoading) &&
+            <div className='connect-projects__import-list-item'>
+              <LoadingSpinner message={t('Loading imported projects')} />
+            </div>
+          }
+
+          {!this.state.isLoading && this.state.attachedSources.length == 0 &&
+            <li className='connect-projects__import-list-item--no-imports'>
+              {t('No data imported')}
+            </li>
+          }
+
+          {!this.state.isLoading && this.state.attachedSources.length > 0 &&
+            this.state.attachedSources.map((item, n) => {
+              return (
+                <li key={n} className='connect-projects__import-list-item'>
+                  <i className='k-icon k-icon-check'/>
+
+                  <div className='connect-projects__import-labels'>
+                    <span className='connect-projects__import-labels-filename'>
+                      {item.filename}
+                    </span>
+
+                    <span className='connect-projects__import-labels-source'>
+                      {item.sourceName}
+                    </span>
+                  </div>
+
+                  <div className='connect-projects__import-options'>
+                    <bem.KoboLightButton
+                      m={['red', 'icon-only']}
+                      onClick={() => this.onRemoveAttachment(item.attachmentUrl)}
+                    >
+                      <i className='k-icon k-icon-trash'/>
+                    </bem.KoboLightButton>
+
+                    <bem.KoboLightButton
+                      m={['blue', 'icon-only']}
+                      onClick={() => this.showColumnFilterModal(
+                        this.props.asset,
+                        {
+                          uid: item.sourceUid,
+                          name: item.sourceName,
+                          url: item.sourceUrl,
+                        },
+                        item.filename,
+                        item.linkedFields,
+                        item.attachmentUrl,
+                      )}
+                    >
+                      <i className='k-icon k-icon-settings'/>
+                    </bem.KoboLightButton>
+                  </div>
+                </li>
+              );
+            })
+          }
+        </ul>
+      </div>
+    );
+  }
+
+  render() {
+    return (
+      <bem.FormView__row>
+        {/* Enable data sharing */}
+        <bem.FormView__cell m={['page-title']}>
+          <i className="k-icon k-icon-folder-out"/>
+          <h2>{t('Share data with other project forms')}</h2>
+        </bem.FormView__cell>
+
+        <bem.FormView__cell m={['box', 'padding']}>
+          <bem.FormView__form>
+            <span>
+              {t('Enable data sharing to allow other forms to import and use dynamic data from this project. Learn more about dynamic data attachments')}
+              &nbsp;
+              <a href='#'>{t('here')}</a>
```

Comment:
The url is still missing. I think it's going to be `dynamic_data_attachment.html` - do it similarly to `AUDIT_SUPPORT_URL`